interop/src/server.rs

```diff
@@ -172,6 +172,9 @@ pub struct EchoHeadersSvc<S> {
 
 impl<S: NamedService> NamedService for EchoHeadersSvc<S> {
     const NAME: &'static str = S::NAME;
+    fn grpc_method(path: &str) -> Option<tonic::GrpcMethod<'static>> {
```

Comment:
What is the goal of this fn? Why can't we just inject a gRPC method into the Request object passed to server handlers?

Reply:
Using it can easily retrieve the static gRPC method info via the HTTP URL path. Injecting this info into the Request object is hard to make it happen if we want to keep it in a static lifetime. @LucioFranco